bin/configure-app.sh

```diff
@@ -214,15 +210,19 @@ function ConfigureWASB() {
   LogInfo "Configuring WASB"
   CheckValueSetOrExit "WASB service name" "$wasb_service_name"
   CheckValueSetOrExit "WASB Host" "$wasb_host"
-  CheckValueSetOrExit "WASB Shared Access Signature token ID" "$wasb_sas_token_id"
+  CheckValueSetOrExit "WASB Shared Access Signature token" "$wasb_sas_token"
 
   jq ".webapp.storageProtocol = \"wasbs\" |
     .hdfs.enabled = false |
     .azure.wasb.enabled = true |
+    .azure.wasb.fetchSasTokensFromKeyVault = false |
     .azure.wasb.defaultStore.blobHost = \"$wasb_host\" |
     .azure.wasb.defaultStore.container = \"$wasb_container\" |
-    .azure.wasb.defaultStore.sasTokenId = \"$wasb_sas_token_id\"" \
+    .azure.wasb.defaultStore.sasToken = \"$wasb_sas_token\"" \
     "$triconf" | sponge "$triconf"
+
+  # Not really sure this is required but it's probably safest to ensure wasbs
+  sed -i 's@wasb://@wasbs://@g' "/opt/trifacta/conf/hadoop-site/core-site.xml"
```

Comment:
Move from `ConfigurePostgres` to a new `ConfigureCoreSite`?

Reply:
I think that's an artifact of the diff display, it's actually in `ConfigureWASB` which I think makes sense.